### PR TITLE
Fixed code

### PR DESCRIPTION
**Pull request by dsliusarchyk**
done

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,32 @@
...
+  background-repeat: no-repeat;
+}
+
+/* Активні зірки для модифікатора --N */
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('./images/star-active.svg'); /* Зображення активної зірки */
```
Comment (VitaliyBondarenko1982) [inline src/style.css:31]: do you really needs comments?
left useless comment is a bad practice, made code not clean and can become wrong at a time.
code should be selfcommented (understandable without comments).
If you need comments - general practice to write them in english.

File: src/index.html [hunk 2]
```diff
@@ -14,5 +14,46 @@
   </head>
   <body>
     <h1>Stars</h1>
```
Comment (VitaliyBondarenko1982) [inline src/index.html:16]: remobe h1

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Good job.
Please check comments and add demo links to PR description

File: src/style.css [hunk 3]
```diff
@@ -1 +1,32 @@
...
+
+/* Стиль для кожної зірки */
+.stars__star {
+  width: 16px; /* Розмір зірки */
+  height: 16px; /* Висота зірки */
+  margin-right: 4px; /* Відстань між зірками */
+  background-image: url('./images/star.svg'); /* Зображення порожньої зірки */
+  background-size: contain;
+  background-repeat: no-repeat;
+}
```
Comment (vadiimvooo) [inline src/style.css:23]:
```suggestion
.stars__star {
  width: 16px;
  height: 16px;
  margin-right: 4px;
  background-image: url('./images/star.svg');
  background-position: center;
  background-repeat: no-repeat;
}
```

File: src/style.css [hunk 4]
```diff
@@ -1 +1,32 @@
...
+div,
+span {
+  margin: 0;
+  padding: 0;
+}
+
+/* Загальний стиль блоку зірок */
+.stars {
+  display: flex;
+  margin-bottom: 10px; /* Відступ між блоками */
```
Comment (vadiimvooo) [inline src/style.css:12]:
```suggestion
```

**Review by vadiimvooo — CHANGES_REQUESTED** [review]
Seems like there are no changes from last review. Please feel free to ask for help in fe_chat to fix the comments.

File: readme.md [hunk 5]
```diff
@@ -22,8 +23,8 @@ Hold `Alt` key (`Option` on MacOS) to measure distances in Figma.
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://DenScriptor.github.io/layout_stars/)
+- [TEST REPORT LINK](https://DenScriptor.github.io/layout_stars/report/html_report/)
```
Comment (volodymyr-soltys97) [inline readme.md:27]: Add these links to the PR description 
image

File: src/style.css [hunk 6]
```diff
@@ -1 +1,38 @@
...
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+
+
+.stars--0 .stars__star:nth-child(n) {
+  background-image: url('./images/star.svg'); 
+}
```
Comment (volodymyr-soltys97) [inline src/style.css:30]:
```suggestion
}

```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, your demo link isn't working, you need to make a deploy and check it after demo, if you need some help pls feel free in the fe_chat
image

File: src/style.css (hunk 6 above)
Comment (etojeDenys) [inline src/style.css:30]: we don't need that

File: readme.md (hunk 5 above)
Comment (etojeDenys) [reply readme.md:27]: feel free to ask for some help in the chat how to fix them

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 6 above)
Comment (etojeDenys) [reply src/style.css:30]: image

**Review by etojeDenys — CHANGES_REQUESTED** [review]
don't forget to ask for some help with those links

File: src/style.css (hunk 6 above)
Comment (vadiimvooo) [inline src/style.css:30]: You have to remove this styles as it was metioned before.

**Review by vadiimvooo — CHANGES_REQUESTED** [review]
Almost done. Please dont hesitate to ask for help in fe_chat.

**Review by vadiimvooo — CHANGES_REQUESTED** [review]
.

**Comment by dsliusarchyk** [comment]
Безіменний
I don't understand anything, here's what my code looks like

File: src/index.html [hunk 7]
```diff
@@ -13,6 +13,60 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stats-block">
+      <h2>Statistics</h2>
+      <p>Total reviews: 5</p>
+      <p>Average rating: 4.5</p>
+    </div>
```
Comment (etojeDenys) [inline src/index.html:20]: you don't need it here

File: src/style.css [hunk 8]
```diff
@@ -1 +1,39 @@
...
+  font-family: Arial, sans-serif;
+  text-align: center;
+  background-color: #f9f9f9;
+  color: #333;
+}
+
+.stats-block {
+  text-align: center;
+  margin-bottom: 20px;
+}
```
Comment (etojeDenys) [inline src/style.css:13]: remove it too

File: src/style.css [hunk 9]
```diff
@@ -1 +1,39 @@
...
+
+.stars {
+  display: flex;
+  justify-content: center;
+  gap: 4px;
+  margin: 10px auto;
+  margin-bottom: 15px;
+  align-items: center;
```
Comment (etojeDenys) [inline src/style.css:21]:
```suggestion
  display: flex;
  flex-direction: row;
  width: fit-content;
```

File: src/style.css [hunk 10]
```diff
@@ -1 +1,39 @@
...
+}
+
+.stars__star {
+  width: 24px;
+  height: 24px;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url('./images/star.svg');
+}
```
Comment (etojeDenys) [inline src/style.css:31]:
```suggestion
.stars__star {
  width: 16px;
  height: 16px;
  background-position: center;
  background-repeat: no-repeat;
  margin-right: 4px;
}
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Comment by dsliusarchyk** [comment]
пройшов тест, все працює, але не можу відкрити новий PR :(
прохання перевірити код
https://github.com/dsliusarchyk/layout_stars.git

**Pull request closed without merging**